tests/volume.postman_environment.json

```diff
@@ -1,54 +0,0 @@
-{
```

Comment:
Don't we need this file for newman?

Reply:
We do not actually, the environment information is contained within the Volume Testing.postman_collection.json. The environment file is just good documentation and can be used to pass off to other devs.